src/lib/__tests__/server-utilities.test.js

```diff
@@ -0,0 +1,18 @@
+import { md5Hash } from 'lib/server-utilities';
+jest.unmock('lib/server-utilities');
+
+describe('md5Hash', () => {
+  const adminClientHash = md5Hash('./static/build/editor-client.js');
+  it('returns a string', () => {
+    expect(typeof adminClientHash).toBe('string');
+  });
+
+  it('hashes consistently', () => {
+    const secondAdminClientHash = md5Hash('./static/build/editor-client.js');
+    expect(adminClientHash).toBe(secondAdminClientHash);
+  });
```

Comment:
Well I'm just trying to start some good form testing slowly, and a hash function is supposed to do 2 things.
1. Not collide / not have the same hash for different files
2. Always have the same hash for the same file.

So I'm simply testing the second one as the first one is pretty impossible to test.
> but I can't imagine this happening.

Of course not if the function is implemented correctly, but that's why we test right? To make sure it's implemented correctly :). I know we use a library for it all here, but this will also implicitly test that everything works as it should btw, for example during our linting stuff if someone had renamed some variables incorrectly so the function broke this test would have probably caught it :).